docs/server/source/production-deployment-template/node-config-map-and-secrets.rst

```diff
@@ -220,10 +220,9 @@ portal using your admin account, click **APIs** and click on **Integration**
 for the relevant API.
 Scroll to the bottom of the page and click the small link
 in the lower right corner, labelled **Download the NGINX Config files**.
-You'll get a ``.zip`` file.
```

Comment:
I just talked with him about this. We'll have to figure out how to get the NGINX config file, Lua stuff and the various 3scale secrets in the future. This page has some tips:

https://access.redhat.com/articles/3074711